tools/testdata-scripts/locales.rs.data

```diff
@@ -57,6 +57,4 @@ const LOCALES: &[LanguageIdentifier] = &[
     // Turkish:
     // - Interesting case-mappings
     langid!("tr"),
-    // Root data
-    langid!("und"),
```

Comment:
Question: Where does `und` get added back if you remove it from here?

Reply:
`und` gets included automatically by datagen if you include at least one language: https://github.com/unicode-org/icu4x/blob/main/provider/datagen/src/lib.rs#L371

Removing it here removes it from `icu_tesdata::locales()` though. I'm undecided whether it should be exposed there, as it's mainly there for fallback.

Reply:
OK, I see.

The line you point to was intended to prevent the footgun of "forgetting" to include `und` when running commands such as `icu4x-datagen --locales en,es,zh`. However, that is unlikely to be good long-term behavior, and #2683 should clean this up by automatically including parents (and offering additional modes for choosing which locales to include).

`und` is a fine, valid locale. It's the equivalent of the 001 region code.

Therefore I believe the correct behavior is to keep `und` in locale lists.

Reply:
Reverted